python/ray/dataframe/concat.py

```diff
@@ -1,90 +1,125 @@
-import pandas as pd
-import numpy as np
-from .dataframe import DataFrame as rdf
-from .utils import (
-    from_pandas,
-    _deploy_func)
-from functools import reduce
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pandas
+from .dataframe import DataFrame
+from .utils import _reindex_helper
 
 
 def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
            keys=None, levels=None, names=None, verify_integrity=False,
            copy=True):
 
-    def _concat(frame1, frame2):
-        # Check type on objects
-        # Case 1: Both are Pandas DF
-        if isinstance(frame1, pd.DataFrame) and \
-           isinstance(frame2, pd.DataFrame):
-
-            return pd.concat((frame1, frame2), axis, join, join_axes,
+    if keys is not None:
+        objs = [objs[k] for k in keys]
+    else:
+        objs = list(objs)
+
+    if len(objs) == 0:
```

Comment:
Need to handle case of `ValueError: All objects passed were None`

Reply:
This is what I see when all are `None`:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-f28cab36205e> in <module>()
----> 1 pd.concat([None])

~/software_builds/ray/python/ray/dataframe/concat.py in concat(objs, axis, join, join_axes, ignore_index, keys, levels, names, verify_integrity, copy)
     20 
     21     if len(objs) == 0:
---> 22         raise ValueError("No objects to concatenate")
     23 
     24     try:

ValueError: No objects to concatenate
```

Reply:
Resolved below